CMakeLists.txt

```diff
@@ -241,6 +244,8 @@ message(STATUS "NGRAPH_MLIR_ENABLE:                   ${NGRAPH_MLIR_ENABLE}")
 message(STATUS "NGRAPH_NATIVE_ARCH_ENABLE:            ${NGRAPH_NATIVE_ARCH_ENABLE}")
 message(STATUS "NGRAPH_NOP_ENABLE:                    ${NGRAPH_NOP_ENABLE}")
 message(STATUS "NGRAPH_ONNX_IMPORT_ENABLE:            ${NGRAPH_ONNX_IMPORT_ENABLE}")
+message(STATUS "NGRAPH_INFERENCE_ENGINE_ENABLE:       ${NGRAPH_INFERENCE_ENGINE_ENABLE}")
```

Comment:
This list is in alphabetical order. Please move these lines to the correct position in the list.